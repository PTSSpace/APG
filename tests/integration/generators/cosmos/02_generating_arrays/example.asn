Module-example DEFINITIONS AUTOMATIC TAGS ::= BEGIN
    Digits ::= INTEGER(0..9)

    Example ::= SEQUENCE {
        zip-code SEQUENCE (SIZE (5)) OF Digits
    }
END
