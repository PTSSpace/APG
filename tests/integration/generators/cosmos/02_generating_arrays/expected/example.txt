# This file was autogenerated from ASN.1 model.

TELEMETRY example Example BIG_ENDIAN ""
    APPEND_ARRAY_ITEM   zip-code                              4 UINT         20 ""
